Lib/test/test_zipfile.py

```diff
@@ -773,6 +773,189 @@ def test_too_many_files_append(self):
             self.assertEqual(content, "%d" % (i**3 % 57))
         zipf2.close()
 
+    def make_zip64_file(
```

Comment:
I would add tests for valid zip64 files: for which `xxx_size_64_set == xxx_size_extra`.